libcontainer/integration/exec_test.go

```diff
@@ -1048,25 +1049,44 @@ func TestHook(t *testing.T) {
 	if testing.Short() {
 		return
 	}
-	root, err := newTestRoot()
+
+	bundle, err := newTestBundle()
 	ok(t, err)
-	defer os.RemoveAll(root)
+	defer remove(bundle)
 
 	rootfs, err := newRootfs()
 	ok(t, err)
 	defer remove(rootfs)
 
 	config := newTemplateConfig(rootfs)
-	expectedBundlePath := "/path/to/bundle/path"
+	expectedBundlePath := bundle
 	config.Labels = append(config.Labels, fmt.Sprintf("bundle=%s", expectedBundlePath))
+
+	getRootfsFromBundle := func(bundle string) (string, error) {
```

Comment:
Because this is a temporary function which is only used by `TestHook`, so I thought it's not worth a standalone function, and `utils_test.go` should contain common functions.  :-)